Atlassian.Jira/IssueType.cs

```diff
@@ -54,7 +54,7 @@ protected override async Task<IEnumerable<JiraNamedEntity>> GetEntitiesAsync(Jir
             {
                 // There are multiple issue types with the same name. Possibly because there are a combination
                 //  of classic and NextGen projects in Jira. Get the issue types from the project if it is defined.
-                results = await jira.IssueTypes.GetIssueTypesForProjectAsync(ProjectKey).ConfigureAwait(false);
+                results = (await jira.Projects.GetProjectIssueTypesAndComponentsAsync(ProjectKey, token).ConfigureAwait(false)).Item1;

```

Comment:
if you name the items in the response type we could use something better than `Item1` :)